server.js

```diff
@@ -25,10 +47,69 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
+
+//Start page 
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({
+  Message: "Avocado sale data", 
+  Routes: [{ 
+    "/avocadoSales": "all of the data",
+    "/avocadoSales/:date": "example: /avocadoSales/2015-10-25",
+    "/highestPrice": "Sorts the data in order of highest price",
+    "/avocados/:id": "example: /avocadoSales/id/63834553a60461fc8065b2bf"
+  }]
+ 
+  });
+});
+
+// All the data
+app.get("/avocadoSales", async (req, res) => {
+  const allTheAvocadoSales =  await AvocadoSale.find({});
+  res.status(200).json({
+    success: true,
+    body: allTheAvocadoSales
+  })
+}); 
+
+// All the data sorted by highest average price
+app.get("/avocadoSales/highestPrice", async (req, res) => {
+  const highestPriceAvocadoSale = await AvocadoSale.find({}).sort({averagePrice: -1})
+  res.status(200).json({
+    success: true, 
+    body: highestPriceAvocadoSale
+  })
```

Comment:
Like the comment, makes it clear what you're trying to do.

And this is what you showed during demo, nicley done 😄